mock-server/index.js

```diff
@@ -0,0 +1,33 @@
+var express = require('express');
+var app = express();
+var expressWs = require('express-ws')(app);
+
+app.use(function (req, res, next) {
+  console.log('middleware');
+  req.testing = 'testing';
+  return next();
+});
+
+app.get('/', function (req, res, next) {
+  console.log('get route', req.testing);
+  res.end();
+});
+
+app.ws('/', function (ws, req) {
+  ws.on('message', function (msg) {
+    console.log(msg);
+  });
+  ws.on('close', function () {
+    console.log('disconnection');
+  });
+  console.log('socket', req.testing);
+});
+
+setInterval(function () {
+  expressWs.getWss('/').clients.forEach(function (client) {
```

Comment:
The websocket path is wrong. Here you can see the proper path `/metrics`. https://github.com/frees-io/freestyle-opscenter/blob/master/core/src/main/scala/runtime/endpoints.scala#L77

Reply:
Addressed in 45e5213cd806a471501424b96d31662387895ae5